core/domain/topic_domain.py

```diff
@@ -1979,8 +1973,7 @@ def to_dict(self) -> TopicRightsDict:
         """
         return {
             'topic_id': self.id,
-            'manager_names': user_services.get_human_readable_user_ids( # type: ignore[no-untyped-call]
-                self.manager_ids),
+            'manager_ids': self.manager_ids,
```

Comment:
Should we fix the places where `to_dict` is used?

Reply:
`to_dict` is not really called anywhere. I have confirmed by putting up logs.

Reply:
Hmm, can we remove this function then? If it is not used.

Reply:
yes can be removed. PTAL @vojtechjelinek